tests/unit_tests/weightwindows/test.py

```diff
@@ -241,7 +241,20 @@ def test_roundtrip(run_in_tmpdir, model, wws):
         assert(ww_out == ww_in)
 
 
-def test_ww_attrs(run_in_tmpdir, model):
+def test_ww_attrs_python(model):
```

Comment:
what is this testing? we set `_energy_bounds = None` in the constructor so unless we pass it in, we're guaranteed to get it back no?

Reply:
This is testing a that failure from the state of the code before the PR has been resolved. I can add a comment to that effect if you like.